tmt/steps/provision/testcloud.py

```diff
@@ -92,10 +92,10 @@ def import_testcloud():
       <target dev='vdb' bus='virtio'/>
       <address type='pci' domain='0x0000' bus='0x00' slot='0x08' function='0x0'/>
     </disk>
-    <interface type='network'>
-        <mac address="{{ mac_address }}"/>
-      <source network='default'/>
-      <model type='rtl8139'/>
+    <interface type='user'>
+      <mac address="{{ mac_address }}"/>
+      <ip family='ipv4' address='172.17.2.0' prefix='24'/>
```

Comment:
for some multihost test, not sure how much it is implemented already in TMT, but would be possible to have there tests, that machines are able to communicate together. If using same network prefix under `user` network can communicate to each other. I think yes, but not sure.

Reply:
I am not sure if it can be done with testcloud as it is right now, it could be achieved by implementing socket networking in testcloud imo. I think we can add it to the roadmap and implement that later as a separate PR?

Reply:
Yes, makes sense to handle this as a separate pull request. Currently there is no support for multihost testing in `tmt`.

Reply:
Yes, agreed, I was just not sure if there is any implementation for multihost in TMT right now.